### PR TITLE
[show][platform][tests] Fix and skip multi ASIC 'show platform' tests

### PR DESCRIPTION
**Pull request by smaheshm**
### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [ x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911

#### What is the motivation for this PR?
'show platform' tests were failing for multi ASIC. Fixed one, skipped for others that are not supported yet.

#### How did you do it?
Add an exemption for 'platform_api_server' for multi ASIC platform running 201911.

#### How did you verify/test it?
```
samaddik@svcstr-server-2:/var/sonic-mgmt/tests$  pytest platform_tests/cli/test_show_platform.py  --testbed=vmsvc1-t1-nmasic-acs-1 --inventory=../ansible/strsvc,../ansible/veos  --testbed_file=../ansible/testbed.csv --host-pattern=svcstr-nmasic-acs-1  --module-path=../ansible/library --skip_sanity
/usr/local/lib/python2.7/dist-packages/ansible/parsing/vault/__init__.py:44: CryptographyDeprecationWarning: Python 2 is no longer supported by the Python core team. Support for it is now deprecated in cryptography, and will be removed in the next release.
  from cryptography.exceptions import InvalidSignature
============================================================================================================ test session starts ============================================================================================================
platform linux2 -- Python 2.7.17, pytest-4.6.5, py-1.10.0, pluggy-0.13.1
ansible: 2.8.12
rootdir: /var/sonic-mgmt/tests, inifile: pytest.ini
plugins: html-1.22.1, repeat-0.9.1, forked-1.3.0, xdist-1.28.0, metadata-1.11.0, ansible-2.2.2
collecting ... ['conf-name', 'group-name', 'topo', 'ptf_image_name', 'ptf', 'ptf_ip', 'ptf_ipv6', 'server', 'vm_base', 'dut', 'inv_name', 'auto_recover', 'comment']
Finished testbed info generating.
/usr/local/lib/python2.7/dist-packages/ansible/parsing/vault/__init__.py:44: CryptographyDeprecationWarning: Python 2 is no longer supported by the Python core team. Support for it is now deprecated in cryptography, and will be removed in the next release.
  from cryptography.exceptions import InvalidSignature
collected 8 items

platform_tests/cli/test_show_platform.py ...s.s.s                                                                                                                                                                                     [100%]

============================================================================================================= warnings summary ==============================================================================================================
/usr/local/lib/python2.7/dist-packages/_pytest/config/__init__.py:538
  /usr/local/lib/python2.7/dist-packages/_pytest/config/__init__.py:538: PytestAssertRewriteWarning: Module already imported so cannot be rewritten: tests.common.dualtor
    self.import_plugin(import_spec)

-- Docs: https://docs.pytest.org/en/latest/warnings.html
============================================================================================= 5 passed, 3 skipped, 1 warnings in 32.52 seconds ==============================================================================================
s
```

### CODE REVIEW

File: tests/common/devices/sonic.py [hunk 1]
```diff
@@ -762,6 +762,9 @@ def get_pmon_daemon_states(self):
                 else:
                     logging.debug("Daemon %s is disabled" % key)
                     exemptions.append(key)
+
+            if self.is_multi_asic and self.sonic_release in ['201911']:
```
Comment (judyjoseph) [inline tests/common/devices/sonic.py:766]: Should we add platform_api_server as a common exemption? without checking for release/asic.
This is not really a pmon daemon, it is a sever spawned to test the platform api's.

Comment (smaheshm) [reply tests/common/devices/sonic.py:766]: I don't know about single ASIC, @sujinmkang  can you comment on this question.

**Review by judyjoseph — APPROVED** [review]

**Pull request merged**